### ISSUE TITLE
Errorcode=-3 when downgrade ipad2,2

### ISSUE THREAD

**Issue by Danfrid** [tihmstar/futurerestore#150]
![2017-07-28 22-57-07](https://user-images.githubusercontent.com/4591705/28734176-54ec5524-73e8-11e7-83e4-27c9c158ea95.png)
What's the matter? 
shsh and ipsw in stock..

**Comment by 2Klasic** [comment]
the device needs to be in a pwndfu state if not future restore will all show that error

**Comment by tihmstar** [comment]
I think on ubuntu you might need to run the tool as root (sudo).

**Comment by Danfrid** [comment]
Of course, I try in kdfu (using enterkdfu tool) and had try sudo, but useless :\

**Comment by 2Klasic** [comment]
if you attempt to use enterkdfu tool please make certain you have valid pwnedibss because kloader will kick the device into dfu mode but it wont be seen by your computer if the pwnedibss is not valid..i would recommend using tihmstar version of the kdfu application from his repo repo.tiihmstar.net.

**Comment by Sn0wCooder** [comment]
@2Klasic My pwnediBSSs are extracted with Odysseus .-.

**Comment by Danfrid** [comment]
@tihmstar can U explain in what problem?
danfr1d@danfr1d-Lenovo-B50-30:~/Документы/3$ sudo ./f --use-pwndfu -t 9.3.2.shsh --latest-baseband 9.3.2.ipsw 
Version: b99eb8140d8e6c23f34e950102bb79e61c72384d - 152
Libipatcher Version: a6cc5cfa8d421987d06f6157130ef3c75391efa4 - 39
Odysseus Support: yes
[INFO] 32bit device detected
futurerestore init done
reading ticket 9.3.2.shsh done
user specified to use latest signed baseband (WARNING, THIS CAN CAUSE A NON-WORKING RESTORE)
[TSSC] opening firmware.json
[DOWN] downloading file https://api.ipsw.me/v2.1/firmwares.json/condensed
[TSSC] selecting latest iOS: 9.3.5
[TSSC] got firmwareurl for iOS 9.3.5 build 13G36
100 [===================================================================>]
downloading Baseband

user specified to use latest signed baseband (WARNING, THIS CAN CAUSE A NON-WORKING RESTORE)
[TSSC] opening firmware.json
[DOWN] downloading file https://api.ipsw.me/v2.1/firmwares.json/condensed
[TSSC] selecting latest iOS: 9.3.5
[TSSC] got firmwareurl for iOS 9.3.5 build 13G36
100 [===================================================================>]
downloading Baseband
100 [===================================================================>]
[WARNING] failed to read BasebandGoldCertID from device! Is it already in recovery?
[WARNING] using tsschecker's fallback to get BasebandGoldCertID. This might result in invalid baseband signing status information
[TSSC] opening /tmp/futurerestore/basebandManifest.plist
[TSSR] User specified to request only a Baseband ticket.
ERROR: Unable to get BasebandFirmware node
ERROR: Unable to find required BbGoldCertId in parameters
Request URL set to https://gs.apple.com/TSS/controller?action=2
TSS server returned: STATUS=49&MESSAGE=An internal error occurred.
ERROR: TSS request failed (status=49, message=An internal error occurred.)
Sending TSS request attempt 1... [Error] baseband firmware isn't signed
Failed with errorcode=-3

okay.. probably now problem with BbGoldCertId, which about U wrote on your blog)
I don't know, will it help to solve this problem, but:        BasebandCertId:  257
p.s. I dumped onboard 9.3.2.shsh with odysseus, which doesn't contain ibss and ibec, is that normal?
Only LLB and apticket

**Comment by EngLearnsh** [comment]
I got the totally same result as you, and i really don't know how to solve it. Do u have any solutions for it?

**Comment by Danfrid** [comment]
@EngLearnsh what is your firmware version now?

**Comment by EngLearnsh** [comment]
Sorry for my late response. Now i am using 8.4.1 because there is only a few softwares which still adaptes 6.1.3 and i donot wanna use compatible version. So I wanna down to 7.1.2, it's smooth than 8 and 9 and most softs adapt it. Btw my device is same as you, iPad2,2, and i tried downgrading both on 6.1.3 and 9.3.5 but it just stuck right on bbgoldcertid :p

**Comment by EngLearnsh** [comment]
i think @tihmstar just forget to add that id which related to our iPad2,2, so.....keeping waiting?

**Comment by Danfrid** [comment]
@EngLearnsh yeah, let's hope for adding

**Comment by EngLearnsh** [comment]
me too

**Comment by davidsteglich** [comment]
@Danfrid @EngLearnsh You need an older version of futurerestore. This is a bug which came when he updated futurerestore for ios 10, but I think he deleted the older versions. If you found an older version pls send me a link :D

**Comment by Danfrid** [comment]
@davidsteglich https://github.com/tihmstar/futurerestore/issues/207
check it)
